packages/react-data-grid/src/Cell.tsx

```diff
@@ -159,31 +160,96 @@ export default class Cell<R> extends React.Component<CellProps<R>> implements Ce
     return allEvents;
   }
 
+  handleDeleteSubRow = (): void => {
+    const { idx, rowIdx, expandableOptions, cellMetaData, rowData } = this.props;
+
+    const { onDeleteSubRow } = cellMetaData;
+
+    if (onDeleteSubRow) {
+      onDeleteSubRow({
+        idx,
+        rowIdx,
+        rowData,
+        expandArgs: expandableOptions
+      });
+    }
+  }
+
   render() {
-    const { idx, rowIdx, column, value, tooltip, children, height, cellControls, expandableOptions, cellMetaData, rowData, isScrolling } = this.props;
+    const { rowIdx, column, value, tooltip, children, height, cellControls, expandableOptions, cellMetaData, rowData, isScrolling, isSummaryRow } = this.props;
     if (column.hidden) {
       return null;
     }
 
     const style = this.getStyle();
-    const className = this.getCellClass();
-    const cellContent = children || (
-      <CellContent<R>
-        idx={idx}
-        rowIdx={rowIdx}
-        column={column}
-        rowData={rowData}
-        value={value}
-        tooltip={tooltip}
-        expandableOptions={expandableOptions}
-        height={height}
-        onDeleteSubRow={cellMetaData.onDeleteSubRow}
-        cellControls={cellControls}
-        isScrolling={isScrolling}
-        isRowSelected={this.props.isRowSelected}
-        onRowSelectionChange={this.props.onRowSelectionChange}
+    const cellClassName = this.getCellClass();
+    const cellValueClassName = classNames('react-grid-Cell__value', { 'cell-tooltip': !!tooltip });
+
+    if (isSummaryRow) {
+      return (
+        <div
+          ref={this.cell}
+          className={cellClassName}
+          style={style}
+        >
+          <div className={cellValueClassName}>
+            <div className="react-grid-Cell__container">
+              <span>
```

Comment:
We can remove the span?

Reply:
I want to remove the span, but I am not sure if we should do it in this PR? Maybe in another cell refactor PR?